docs/api/rtk-query/createApi.mdx

```diff
@@ -237,7 +234,98 @@ export const { endpoints, reducerPath, reducer, middleware } = api
 // see `createApi` overview for _all exports_
 ```
 
-#### Transforming the data returned by an endpoint before caching
+### `tagTypes`
+
+[summary](docblock://query/createApi.ts?token=CreateApiOptions.tagTypes)
+
+[examples](docblock://query/createApi.ts?token=CreateApiOptions.tagTypes)
+
+### `reducerPath`
+
+[summary](docblock://query/createApi.ts?token=CreateApiOptions.reducerPath)
+
+[examples](docblock://query/createApi.ts?token=CreateApiOptions.reducerPath)
+
+### `serializeQueryArgs`
+
+[summary](docblock://query/createApi.ts?token=CreateApiOptions.serializeQueryArgs)
+
+Defaults to:
+
+```ts no-transpile
```

Comment:
:question: Do we really need to copy the entire implementation here? Seems like a written description of behavior would be sufficient.

Reply:
Hmm I agree. A written description would certainly cover it.

However, can you think of a theoretical example for a custom version of `serializeQueryArgs` someone might want to use rather than the default? If so, I think that would be beneficial as a snippet

Reply:
FYI I've replaced this with a written description, but I can't think of a realistic example someone would use to override it